lighthouse-core/gather/gather-runner.js

```diff
@@ -165,6 +166,20 @@ class GatherRunner {
     } else if (mainRecord.hasErrorStatusCode()) {
       errorDef = {...LHError.errors.ERRORED_DOCUMENT_REQUEST};
       errorDef.message += ` Status code: ${mainRecord.statusCode}.`;
+    } else if (!mainRecord.finished) {
+      // could be security error
+      const securityState = await driver.getSecurityState();
```

Comment:
super nit: WDYT about sprinkling some nice destructuring here, `const {securityState, explanations} =`? somethin' about `securityState.securityState` just spoke to me :)

Reply:
I didn't even notice that :O